cyhy/mailer/__init__.py

```diff
@@ -1,3 +1,3 @@
 """This package contains the cyhy-mailer code."""
 
-__version__ = "1.7.4"
```

Comment:
Does this require a version bump? The Docker composition files aren't attached to the logic of the Docker image or the Python package which should be what dictates version changes.

Reply:
I'm fine if you want to remove the version bump.

Reply:
I was on the fence, but I went with Shane's suggestion to bump the version.  I don't think it makes much difference either way.

Reply:
@mcdonnnj Can we resolve this conversation?

Reply:
:/ I'm not a fan of bumping versions for no functionality change. If the two of you want to bump it then fine, but those Docker composition files really belong in [cisagov/ansible-role-cyhy-mailer](https://github.com/cisagov/ansible-role-cyhy-mailer) to mirror updates to other, similar Docker setups such as [cisagov/ansible-role-code-gov-update](https://github.com/cisagov/ansible-role-code-gov-update) and [cisagov/ansible-role-vdp-scanner](https://github.com/cisagov/ansible-role-vdp-scanner).

Reply:
My point is the version tracks the Python package, and as a result the Docker image. The Docker composition files just happen to live here and If you download the [Docker image](https://hub.docker.com/r/cisagov/cyhy-mailer) or install this Python package directly these files are nowhere to be found.

Reply:
I'm fine if you want to remove the version bump.

Reply:
The version bump has been removed.